src/proxy/src/main/java/ee/ria/xroad/proxy/protocol/ProxyMessage.java

```diff
@@ -111,8 +112,9 @@ public SoapFault getFault() {
      */
     public String getSoapContentType() {
         return isMimeEncodedSoap() || hasAttachments()
-                ? (originalContentType != null ? originalContentType : encoder.getContentType())
-                : MimeUtils.TEXT_XML_UTF8;
+                ? originalContentType != null
+                    ? originalContentType : encoder.getContentType()
+                : MimeTypes.TEXT_XML_UTF8;
     }
```

Comment:
Well, historically we are using still code lines with length 80 character, but X-Road check style uses currently 120 length and indeed, no wrapping here is needed.  Old habits :). We will revert it.